setup.py

```diff
@@ -11,8 +11,7 @@
       package_data={'pytest_html': ['resources/*']},
       entry_points={'pytest11': ['html = pytest_html.plugin']},
       setup_requires=['setuptools_scm'],
-      install_requires=['ansi2html>=1.1.1',
-                        'pytest>=2.3'],
+      install_requires=['pytest>=2.3'],
```

Comment:
isn't this only needed for testing? You could put it in `tests_require` instead of `install_requires`